swarmzero/sdk_context.py

```diff
@@ -79,6 +79,8 @@ def load_default_config(self):
             "enable_multi_modal": self.config.get("model", "enable_multi_modal", False),
             "sample_prompts": self.config.get("sample_prompts", "prompts", []),
         }
+        defaults["max_iterations"] = self.config.get("max_iterations", default=10)
```

Comment:
it's better to put this inside the `defaults` dictionary

Reply:
i suggest putting `max_iterations` inside `model` and also within each agent specified in the config.

example of multi-agent config - https://github.com/swarmzero/examples/blob/main/swarms/dapp_builer_swarm/app/swarmzero_config.toml